test/dlc_tests/ecs/pytorch/training/test_ecs_pytorch_training.py

```diff
@@ -55,6 +56,47 @@ def test_ecs_pytorch_training_mnist_gpu(gpu_only, ecs_container_instance, pytorc
                                          num_gpus=num_gpus)
 
 
+@pytest.mark.model("resnet18")
+@pytest.mark.parametrize("training_script", [PT_S3_PLUGIN_CMD], indirect=True)
+@pytest.mark.parametrize("ecs_instance_type", ["c5.9xlarge"], indirect=True)
+@pytest.mark.parametrize("ecs_ami", [ECS_AML2_CPU_USWEST2], indirect=True)
+def test_ecs_pytorch_s3_training_cpu(cpu_only, ecs_container_instance, pytorch_training, training_cmd,
```

Comment:
nit: Please rename this to contain the term `s3_plugin` to make the test name clear in reports, and also add `@pytest.mark.integration("pt_s3_plugin")`